001/homework03/hw03.py

```diff
@@ -0,0 +1,196 @@
+import random
+
+def fillListRandomNaturalNumbers(n):
+    targetList = list()
+    for i in range(1, n + 1): 
```

Comment:
Будет то же самое количество итераций, если...
```suggestion
    for i in range(n): 
```